### DISCUSSION TITLE
Collaboration Sudoku

### DISCUSSION THREAD

**Discussion by alexzhaohong** [michel-kraemer/sudocle#729] [Ideas]
Hi Michel thank you for Sudocle.

I do Cracking the Cryptic sudokus with a friend and was wondering if it would be possible to have a collaborative sudoku app, similar to Excalidraw https://excalidraw.com/ https://github.com/excalidraw/excalidraw 

Do you think it would be quick to get a proof of concept using websockets? And do you think this a feature that would best be developed for Sudocle? Is the official Cracking The Cryptic Web App not open source?

**Comment by michel-kraemer** [comment]
Hi Alex! This is definitely an interesting idea! Sudocle is developed with Next.js (https://nextjs.org/), which has a server that could be used to share data between computers. I'm currently exporting static HTML, so there's no server involved, but it would be a great weekend project to investigate how Next.js can be used for this and to implement a prototype. What is your plan? Would you like to implement this yourself or do you want me to do it?

> Is the official Cracking The Cryptic Web App not open source?

I'm afraid not. There is no repository on GitHub (or anywhere else) and it does not have an open source license.
